src/runtime/signal.jl

```diff
@@ -40,21 +40,20 @@ ROCSignal(signal::HSA.Signal) = ROCSignal(Ref(signal))
 
 get_handle(signal::ROCSignal) = signal.signal[].handle
 
-function is_done(signal::ROCSignal)
-    value = HSA.signal_load_scacquire(signal.signal[])
-    value < 1
-end
+load_acquire(signal::ROCSignal) = HSA.signal_load_scacquire(signal.signal[])
```

Comment:
I wonder if we should pull the host signal helpers up from hostcall into the `Runtime` module?